packages/tldraw/src/lib/canvas/TldrawSelectionForeground.tsx

```diff
@@ -477,12 +467,15 @@ export const RotateCornerHandle = function RotateCornerHandle({
 	'data-testid'?: string
 }) {
 	const events = useSelectionEvents(corner)
+	const msg = useTranslation()
+	const label = msg(`handle.rotate.${corner}`)
```

Comment:
I guess this is not typechecked properly?

Reply:
😬 yeah it's not really typechecked b/c of the signature: `msg(id?: Exclude<string, TLUiTranslationKey> | string)`
i think we should move away from this system in general in the coming year.